tool/tbot/config/config_destination.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package config
+
+import (
+	"github.com/gravitational/trace"
+)
+
+type ConfigKind string
+
+const (
+	CONFIG_KIND_SSH ConfigKind = "ssh"
+	CONFIG_KIND_TLS ConfigKind = "tls"
+)
+
+// AllKinds lists all valid config kinds, intended for validation purposes.
+var AllKinds = [...]ConfigKind{CONFIG_KIND_SSH, CONFIG_KIND_TLS}
```

Comment:
Does this need to be exported?

If you use it to validate in other packages, I would consider unexporting this, and creating an exported validation func that reads but does not modify `allKinds`.

It's unlikely to happen, but leaving this exported allows other package to mutate it, which could break things.

Reply:
It's not needed outside the package yet, so I'll just leave it unexported for now, we can always add a validation function like you suggested if needed later.